subsys/net/l2/ethernet/gptp/gptp_messages.c

```diff
@@ -129,10 +129,25 @@ struct net_pkt *gptp_prepare_sync(int port)
 	struct net_buf *frag;
 	struct gptp_hdr *hdr;
 
+#if defined(CONFIG_NET_VLAN)
+	struct net_eth_vlan_hdr *hdr_vlan;
```

Comment:
move that after line 202

Reply:
Hmm, I think that is against the coding style. Meaning we should introduce variables at the beginning of the function.

Reply:
This cannot be done because hdr_vlan is filled later in the function. For that we also need the vlan_enabled flag.